code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -90,6 +90,12 @@
 	message_param = "salutes to %t."
 	hands_use_check = TRUE
 
+/datum/emote/living/headtilt
+	key = "tilt"
+	key_third_person = "tilts"
+	message = "tilts their head."
+	message_AI = "tilts the image on their display."
```

Comment:
There seems to be a 'head tilt' emote already in `code\modules\mob\living\emote.dm`:
https://github.com/pariahstation/Pariah-Station/blob/b083e79b170e80b134e9858d274274b1eae61619/code/modules/mob/living/emote.dm#L410-L413
Maybe add these changes onto that?